content/gophercon-2017/writing-a-successful-gophercon-proposal.md

```diff
@@ -0,0 +1,100 @@
++++
```

Comment:
Is this valid markdown? GitHub seems to be rendering it weird on my end.. 
<img width="810" alt="screen shot 2017-01-13 at 9 22 18 pm" src="https://cloud.githubusercontent.com/assets/13757818/21952348/66ebd512-d9d6-11e6-813d-c96566f9b446.png">